SwiftyContextMenu/ContextMenuActionTableViewCell.swift

```diff
@@ -8,25 +8,86 @@
 import UIKit
 
 class ContextMenuActionTableViewCell: UITableViewCell {
-
+    
     private var rightImageView = UIImageView(frame: CGRect(x: 0, y: 0, width: 25, height: 25))
-
+    private var lightSelectedBackgroundView: UIVisualEffectView!
+    private var darkSelectedBackgroundView: UIView!
+    private var separatorView: ContextMenuSeparatorView!
+    private var style: ContextMenuUserInterfaceStyle = .light {
+        didSet {
+            updateSelectedBackgroundView()
+            separatorView.style = style
```

Comment:
I don't feel very safe with this code, all properties are private so we are in control of their life-cycle but I would use an optional here, just in case :) 
`separatorView?.style = style`

Reply:
You're right we don't how it can be evolve in the future. Safety first.